test-cases/tests/dashboards/e04b-verify-slo-dashboard.md

```diff
@@ -23,14 +24,14 @@ Note: this test should only be performed at a time it will not affect other ongo
 ## Steps
 
 1. Make sure that there is at least one active alert for every panel in Grafana SLO dasboard (e.g. pod_down)
-   1. Make sure rhoam-operator pod is scaled down to 0 pods
+   1. Make sure rhmi-operator pod is scaled down to 0 pods
```

Comment:
is file b not rhoam

Reply:
yep, but the pod is named still (confusingly) rhmi-operator

Reply:
ah right ok